tests/Report/JunitReporterTest.php

```diff
@@ -248,17 +246,21 @@ private function assertJunitXmlSchema($xml)
     {
         $xsdPath = __DIR__.'/../../doc/junit-10.xsd';
 
-        libxml_use_internal_errors(true);
-
         static $errorLevels = array(
             LIBXML_ERR_WARNING => 'Warning',
             LIBXML_ERR_ERROR => 'Error',
             LIBXML_ERR_FATAL => 'Fatal Error',
         );
 
+        $internal = libxml_use_internal_errors(true);
+
         $dom = new \DOMDocument();
 
-        $this->assertTrue($dom->loadXML($xml));
+        $load = $dom->loadXML($xml);
+        if (true !== $load) {
+            libxml_use_internal_errors($internal);
+            $this->assertTrue($load, 'XML loading failed.');
```

Comment:
`assertTrue` is not supposed to mart test as failure. it's not intuitive.
Please switch to fail. if you want to provide reasoning, dump the `$load`.